### PR TITLE
Add centos7 command to start config.service

### PR DESCRIPTION
**Pull request by dyozie**
Will backport to 6X_STABLE and 5X_STABLE

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml [hunk 1]
```diff
@@ -648,7 +648,8 @@ ls -l &lt;cgroup_mount_point&gt;/memory/gpdb</codeblock>
             configure one of the following cgroup service commands in your preferred service
             auto-start tool: <ul>
               <li> Redhat/CentOS 7.x and SuSE11+ systems:
-                <codeblock>sudo systemctl enable cgconfig.service</codeblock>
+                <codeblock>sudo systemctl enable cgconfig.service
```
Comment (lisakowen) [inline gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml]: i think `enable` should enable the service to start on next boot.  the `start` command starts it.  so both should not be needed in the auto-start tool.  but start would be needed if they want to use resource groups before the next reboot.

(may not be necessary to get into this much detail here.)

Comment (dyozie) [reply gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml]: I'm not sure what you mean by "both should not be needed in the auto-start tool" but you're right about what each command does.  I think it's probably best to tweak the wording of this step to indicate that it configures the setting through reboots, and also enables it immediately.  I'll add the similar `service start` command for RHEL/CentOS6 as well.

Comment (dyozie) [reply gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml]: (And of course, the requester of this PR clearly wants us to make the instructions work without a system reboot, which seems reasonable.)

Comment (dyozie) [reply gpdb-doc/dita/admin_guide/workload_mgmt_resgroups.xml]: Nevermind - it looks like the RHEL6 process already has the service started.  I'll just add some text to distinguish it in the commands for 7.

**Pull request merged**